client/dashboard/sites/site-fields.tsx

```diff
@@ -148,3 +158,95 @@ export function MediaStorage( { site }: { site: Site } ) {
 
 	return <span ref={ ref }>{ ! isLoading ? value : <LoadingIndicator label="100%" /> }</span>;
 }
+
+function SiteLaunchNag( { site }: { site: Site } ) {
+	const { recordTracksEvent } = useAnalytics();
+
+	// TODO: We have to fix the obscured focus ring issue as the dataview's field value container
+	// uses `overflow:hidden` to prevent any of the fields from overflowing.
+	return (
+		<>
+			<ComponentViewTracker eventName="calypso_dashboard_sites_site_launch_nag_impression" />
+			<ExternalLink
+				href={ `/home/${ site.slug }` }
+				onClick={ () => {
+					recordTracksEvent( 'calypso_dashboard_sites_site_launch_nag_click' );
+				} }
+			>
+				{ getSiteStatusLabel( site ) }
+			</ExternalLink>
+		</>
+	);
+}
+
+function PlanRenewNag( { site, source }: { site: Site; source: string } ) {
+	const { user } = useAuth();
+	const { recordTracksEvent } = useAnalytics();
+
+	if ( site.site_owner !== user.ID ) {
+		return null;
+	}
+
+	const isTrial = isSitePlanTrial( site );
+
+	return (
+		<>
+			<ComponentViewTracker eventName="calypso_dashboard_sites_plan_renew_nag_impression" />
+			<ExternalLink
+				href={
+					isTrial
+						? `/plans/${ site.slug }`
+						: `/checkout/${ site.slug }/${ site.plan?.product_slug }`
+				}
+				onClick={ () => {
+					recordTracksEvent( 'calypso_dashboard_sites_plan_renew_nag_click', {
+						product_slug: site.plan?.product_slug,
+						source,
+					} );
+				} }
+			>
+				{ isTrial ? __( 'Upgrade' ) : __( 'Renew plan' ) }
+			</ExternalLink>
+		</>
+	);
+}
+
+export function Status( { site }: { site: Site } ) {
+	if ( site.is_deleted ) {
+		return <IneligibleIndicator />;
```

Comment:
Would it be better to display "Deleted" as v1?

Reply:
Oohh good idea, updated!